link/src/test/java/com/stripe/android/link/utils/FakeIntentConfirmationInterceptor.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.link.utils
+
+import com.stripe.android.IntentConfirmationInterceptor
+import com.stripe.android.model.ConfirmPaymentIntentParams
+import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.model.StripeIntent
+import kotlinx.coroutines.channels.Channel
+
+internal class FakeIntentConfirmationInterceptor : IntentConfirmationInterceptor {
+
+    private val channel = Channel<IntentConfirmationInterceptor.NextStep>(capacity = 1)
```

Comment:
This is fine as is, but it can also be useful to use turbines in these situations https://github.com/cashapp/turbine#standalone-turbines